auth/user_mgt.go

```diff
@@ -34,6 +34,12 @@ const (
 	maxLenPayloadCC     = 1000
 	defaultProviderID   = "firebase"
 	idToolkitV1Endpoint = "https://identitytoolkit.googleapis.com/v1"
+
+	// Maximum allowed number of users to batch get at one time.
```

Comment:
Suggest: "Maximum number of users allowed to batch get at a time."

Reply:
Done. I've also applied this to the maxDeleteAccountsBatchSize comment immediately below.